theano/gpuarray/extra_ops.py

```diff
@@ -40,7 +39,6 @@ def get_params(self, node):
 
     def make_node(self, x):
         assert x.type.dtype == 'float32', "Only float32 supported for GpuCumSum"
-        x = as_gpuarray_variable(x, infer_context_name(x))
```

Comment:
Also, like how it is done with the other Ops, can i make context_name an attribute of `GpuCumsum` ?